src/main/java/br/com/tecnodev/entities/category/api/CategoryApiController.java

```diff
@@ -0,0 +1,29 @@
+package br.com.tecnodev.entities.category.api;
+
+import br.com.tecnodev.entities.category.Category;
+import br.com.tecnodev.repository.CategoryRepository;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class CategoryApiController {
+
+    private final CategoryRepository categoryRepository;
+
+    public CategoryApiController(CategoryRepository categoryRepository) {
+        this.categoryRepository = categoryRepository;
+    }
+
+    @GetMapping
+    @RequestMapping(value = "/api/categories", produces = {"application/json", "application/xml"}, consumes = MediaType.ALL_VALUE)
```

Comment:
Aqui não poderia ser só o `@GetMapping` não? Dá erro?

```suggestion
    @GetMapping(value = "/api/categories", produces = {"application/json", "application/xml"}, consumes = MediaType.ALL_VALUE)
```

Reply:
Funciona sim, acho que eu estava fazendo testes e esqueci de mudar.